plugin/Dockerfile

```diff
@@ -1,16 +1,38 @@
-FROM node:16 AS build
+FROM  registry.access.redhat.com/ubi8/nodejs-16 AS build
+
+USER root
+RUN command -v yarn || npm i -g yarn
 
 ADD . /usr/src/app
 WORKDIR /usr/src/app
 RUN yarn install && yarn build
 
-FROM nginx:stable
+FROM registry.access.redhat.com/ubi8/nginx-120
+
+# Add application sources to a directory that the assemble script expects them
+# and set permissions so that the container runs without root access
+USER 0
+
+# Install upgrades security
+#   xz-libs RHSA-2022:4991
+#   urllib3 CVE-2020-26137 CVE-2021-33503  Not available in the ubi8
+#
+RUN yum -y upgrade xz-libs
```

Comment:
This is causing my build to hang. See https://github.com/kiali/openshift-servicemesh-plugin/issues/25 for my suggestions - don't do this yum upgrade if env var DEV_MODE is set to "true".

Reply:
> Just ran this... I see:
> 
> ```
> OpenShift Service Mesh Plugin: Version=[0], Commit=[bbd0136c86ef7b5cc2f4f8cfba131ac99d743e27]
> ```
> 
> The version is wrong here. Should be the VERSION env var.
> 
> I can't explain why the version is showing as "0" - it all looks correct in the dockerfile/makefile. I'm looking into it. It doesn't make sense why this doesn't work.

mmm let me check it.

About slowly....is not my case in my env the dockerfile was build so fast....anyway the "yum" instruction is a temporal solution, they are going to update the image soon